code/style.css

```diff
@@ -1,4 +1,219 @@
-html {
-  background: #0025ff;
-  color: #fff;
+* {
+  margin: 0;
+}
+
+body {
+  background-image: url("https://i.ibb.co/bsjcW0C/61e2ca5537f58.jpg");
+  background-size: cover;
+  background-color: #242321;
+  font-family: "Noto Sans", sans-serif;
+}
+
+header {
+  background-color: rgb(144, 158, 178, 0.3);
+  background-blend-mode: screen;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 20px 0;
+}
+.syzygy-logo img {
+  height: 60px;
+  padding: 5px 10px;
+}
+.syzygy-logo-small {
+  display: none;
+}
+.topnav {
+  height: auto;
+  display: flex;
+  justify-content: space-evenly;
+  text-transform: uppercase;
+  font-size: 15px;
+}
+.topnav a:link {
+  padding: 0 20px;
+  text-decoration: none;
+  color: white;
+}
+.topnav a:visited {
+  color: white;
+}
+.main {
+  width: 80%;
+  margin: auto;
+}
+.big-news {
+  display: grid;
+  grid-gap: 10px;
+  grid-template-columns: 1fr 1fr;
+  background-color: white;
+  padding: 5px;
+  margin: 20px 0;
+}
+.big-news-img {
+  width: 100%;
+  height: 100%;
+  object-fit: cover;
+}
+.big-news-item-2 {
+  padding: 20px 25px;
+}
+.big-news-date-published {
+  font-size: 14px;
+  font-style: italic;
+  color: #b88588;
+}
+h2 {
+  font-size: 28px;
+  font-weight: bold;
+  color: #a4abc6;
+  margin-bottom: 5px;
+}
+.big-news-read-more {
+  display: none;
+}
+.big-news-author {
+  text-transform: uppercase;
+  margin-bottom: 15px;
+  color: #b88588;
+  font-size: 14px;
+}
+.big-news-article {
+  border-radius: 2%;
+  background-color: #f5eded;
+  box-shadow: 5px 5px rgb(164, 171, 198, 0.5);
+  padding: 25px 20px 10px 20px;
+}
+.big-news-text {
+  font-size: 14px;
+  color: #232331;
+  margin-bottom: 15px;
+  text-align: justify;
+}
+.other-news {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr;
+  grid-gap: 10px;
+  margin-bottom: 20px;
+}
+.other-news a {
+  text-decoration: none;
+}
+.other-news > * {
+  background-color: #b88588;
+  border-bottom-left-radius: 2%;
+  border-bottom-right-radius: 2%;
+}
+.other-news-img {
+  width: 100%;
+  height: 300px;
+  object-fit: cover;
+  margin-bottom: 10px;
+}
+h3 {
+  font-size: 18px;
+  font-weight: bold;
+  text-shadow: 0 0 3px #131933;
+  color: white;
+  margin: 0px 20px;
+}
+.other-news-date-published {
+  font-size: 14px;
+  color: black;
+  margin: 10px 20px;
+}
+footer {
+  width: 400px;
+  height: auto;
+  color: #f4b3da;
+  opacity: 0.7;
+  font-size: 14px;
+  padding: 10px;
+  margin-left: 20px;
+}
+@media (min-width: 1024px) {
```

Comment:
I guess (not 100 percent sure) you don't need to defined min-width 1024px in this case. Because you already defined media queries for tablet and mobile below. Your CSS is based on desktop-first, am I right?

Reply:
```@media (min-width: 1024px) ```

Reply:
I should have left a comment here. I wanted to remove the hover effects for mobile and tablet, and since I started with desktop first, easiest way to do it was to add a media query for desktop at the end.